qadi_multi_perspective.py

```
@@ -0,0 +1,299 @@
+    from mad_spark_alt.core import setup_llm_providers
+    from mad_spark_alt.core.intent_detector import IntentDetector, QuestionIntent
+    from mad_spark_alt.core.multi_perspective_orchestrator import (
+        MultiPerspectiveQADIOrchestrator,
+        PerspectiveResult,
+    )
+    from mad_spark_alt.core.terminal_renderer import render_markdown
+except ImportError:
+    # Fallback if package is not installed
+    sys.path.insert(0, str(Path(__file__).parent / "src"))
+    from mad_spark_alt.core import setup_llm_providers
+    from mad_spark_alt.core.intent_detector import IntentDetector, QuestionIntent
```

Ruff (0.12.2)

`mad_spark_alt.core.intent_detector.IntentDetector` imported but unused

Remove unused import: `mad_spark_alt.core.intent_detector.IntentDetector`

(F401)